tox.ini

```diff
@@ -1,7 +1,9 @@
 [tox]
+requires =
+    tox-gh-actions
```

Comment:
Do we need this here since we install it in the actions yaml? No big deal, we can merge without an answer on this, it should be fine

Reply:
It's seems like a good idea to specify the deps for tox to work.

Reply:
it's best to specify tox requires here

Reply:
:+1: